cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1475,6 +1475,2211 @@ declare namespace Deno {
+      type: "UpdateExpression";
+      prefix: boolean;
+      operator: "++" | "--";
+      argument: Expression;
+    }
+
+    /**
+     * Updaate a variable or property.
+     * @category Linter
+     * @experimental
+     */
+    export interface AssignmentExpression extends AstBase {
+      type: "AssignmentExpression";
+      operator:
+        | "&&="
+        | "&="
+        | "**="
+        | "*="
+        | "||="
+        | "|="
+        | "^="
+        | "="
+        | ">>="
+        | ">>>="
+        | "<<="
+        | "-="
+        | "%="
+        | "+="
+        | "??="
+        | "/=";
+      left: Expression;
+      right: Expression;
+    }
+
+    /**
+     * Inline if-statement.
+     * @category Linter
+     * @experimental
+     */
+    export interface ConditionalExpression extends AstBase {
+      type: "ConditionalExpression";
+      test: Expression;
+      consequent: Expression;
+      alternate: Expression;
+    }
+
+    /**
+     * MemberExpression
+     * @category Linter
+     * @experimental
+     */
+    export interface MemberExpression extends AstBase {
+      type: "MemberExpression";
+      optional: boolean;
+      computed: boolean;
+      object: Expression;
+      property: Expression | Identifier | PrivateIdentifier;
+    }
+
+    /**
+     * ChainExpression
+     * @category Linter
+     * @experimental
+     */
+    export interface ChainExpression extends AstBase {
+      type: "ChainExpression";
+      expression:
+        | CallExpression
+        | MemberExpression
+        | TSNonNullExpression;
+    }
+
+    /**
+     * Execute multiple expressions in sequence.
+     * @category Linter
+     * @experimental
+     */
+    export interface SequenceExpression extends AstBase {
+      type: "SequenceExpression";
+      expressions: Expression[];
+    }
+
+    /**
+     * A template literal string.
+     * @category Linter
+     * @experimental
+     */
+    export interface TemplateLiteral extends AstBase {
+      type: "TemplateLiteral";
+      quasis: TemplateElement[];
+      expressions: Expression[];
+    }
+
+    /**
+     * The static portion of a template literal.
+     * @category Linter
+     * @experimental
+     */
+    export interface TemplateElement extends AstBase {
+      type: "TemplateElement";
+      tail: boolean;
+      raw: string;
+      cooked: string;
+    }
+
+    /**
+     * Tagged template expression.
+     * @category Linter
+     * @experimental
+     */
+    export interface TaggedTemplateExpression extends AstBase {
+      type: "TaggedTemplateExpression";
+      tag: Expression;
+      typeArguments: TSTypeParameterInstantiation | undefined;
+      quasi: TemplateLiteral;
+    }
+
+    /**
+     * Pause or resume a generator function.
+     * @category Linter
+     * @experimental
+     */
+    export interface YieldExpression extends AstBase {
+      type: "YieldExpression";
+      delegate: boolean;
+      argument: Expression | null;
```

Comment:
It seems sometimes we use `| undefined` and other times `| null`. Should we consolidate on one or is this for a certain reason?

Reply:
In TSEStree they use `null` as an empty value for all non-type nodes and `undefined` as an empty value for type nodes. It's a little weird.